docs/products/kafka/howto/kafka-klaw.rst

```diff
@@ -0,0 +1,138 @@
+7. Click **Add Environment** and enter the following details: 
+   
+   -  **Environment Name:** Select environment from the drop-down list
+   -  **Select Cluster:** Select the cluster you added from the drop-down list. The bootstrap servers and protocol details are automatically populated 
+   -  **Default Partitions:** Enter the number of partitions based on your requirements. The default value is set to 2
+   -  **Maximum Partitions:** Enter the maximum number of partitions. The default value is set to 2
+   -  **Default Replication factor:** Enter the required replication factor. The default value is set to 1
+   -  **Max Replication factor:** Enter the required max replication factor. The default value is set to 1
+   -  **Topic prefix (optional):** Enter a topic prefix
+   -  **Tenant:** The value is set to default Tenant
+   
+      .. note:: Klaw is multi-tenant. Each tenant can manage topics with their own teams in isolation. Every tenant can have their own set of Apache Kafka® environments, and users of one tenant cannot view/access topics, or ACLS from other tenants. It provides isolation avoiding any security breach. For this topic, I have used the default tenant configuration. For more information, see `Klaw documentation <https://www.klaw-project.io/docs/get-started-kafka#configure-the-cluster-to-sync>`_. 
+
+8. Click **Save**. 
+
+If the connection based on the cluster and environment configuration are **successful**, the connection status displays a **blue thumbs-up icon** on the Environments page in the Klaw web interface.  If it is **unsuccessful**, a **red thumbs-down icon** is displayed. 
+
+For unsuccessful connections, check the authentication protocol configurations. See the :ref:`Additional configurations <klaw-additional-configs>` section for further details. After completing these configurations, check the connection status on the Environments page by clicking the thumbs-down icon to see if the setup was successful.
+
+.. _klaw-additional-configs:
+
+Additional configurations
+-------------------------
+`Klaw <https://www.klaw-project.io/>`_ uses the ``application.properties`` file to configure any application-related properties, such as secret key and authentication protocol configurations. Connecting Aiven for Apache Kafka® with Klaw requires you to perform the following additional configurations in the ``application.properties`` file.
+
+Secret key configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set the value of ``klaw.clusterapi.access.base64.secret`` with a secret key in the form of a Base64 encoded string in the ``application.properties`` file located in the following paths: 
+
+* `klaw/cluter-api/src/main/resources`
+* `klaw/core/src/main/resources`
+
+Configure authentication protocol
+""""""""""""""""""""""""""""""""""
+Klaw supports the following authentication protocols to connect to Apache Kafka® clusters:
```

Comment:
I would say this is general to Klaw and not for Aiven for Apache Kafka and Klaw.
I would just set the two separate options as

* Connect Aiven for Apache Kafka® with Klaw via SSL protocol
* Connect Aiven for Apache Kafka® with Klaw via SASL protocol

Reply:
I would still keep this as part of the "Configure authentication protocol", the reason being this is a part of the "Additional configurations" for SSL in Klaw (and to give more context without jumping to other topics). 
If the protocol configurations are not done correctly, the connection between Klaw and Apache Kafka might not be successful.